bundles/org.openhab.binding.bluetooth.daikinmadoka/src/main/resources/OH-INF/thing/daikinmadoka.xml

```diff
@@ -129,4 +139,28 @@
 		</command>
 	</channel-type>
 
+	<channel-type id="brc1h_eyeBrightness">
+		<item-type>Number</item-type>
+		<label>Eye Illumination Brightness</label>
+		<state min="0" max="19" step="1" readOnly="false"/>
+	</channel-type>
+
+	<channel-type id="brc1h_indoorPowerHours">
+		<item-type>Number</item-type>
+		<label>Number of hours system has been powered up</label>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="brc1h_indoorOperationHours">
+		<item-type>Number</item-type>
+		<label>Number of hours system has been operating</label>
+		<state readOnly="true"/>
+	</channel-type>
```

Comment:
How are these channels different? Also you should use units where possible. For time I suggest using `Number:Time`.

Reply:
They don't report the same information: "power hours" is the number of hour the device has been connected to the electricity network - the "operation hours" are the number of hour the device has been powered on. Will change the time to Number:time.